README.md

```diff
@@ -1 +1,37 @@
-# conference
\ No newline at end of file
+## Conference
+> The Conference project  [![portfolio](https://img.shields.io/badge/Link-Portfolio-<blue>)](https://fahdrahali.github.io/) **This project is responsive with desktop and mobile version, it Contains two pages, a home page and an about page also a featured speackers and main program section.**
+
+## Built with
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo
+[Conference](https://fahdrahali.github.io/)
```

Comment:
- [ ] The live demo link is broken. kindly make sure that your project is deployed and add the link of the online version of your project to your README.